### DISCUSSION TITLE
HAVE BEEN ASKING GITHUB SINCE 2022 WHY OUR DEV ACCOUNT HAS BEEN SUSPENDED, NO RESPONSE?!

### DISCUSSION THREAD

**Discussion by KevinHelton** [community/community#60865] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I Start by saying that I am not a coder.
I am the owner of a project that has used github since 2011.

In 2022 after supporting github for over a decade i cannot access the account.
All i get is the below message—

Access to your account has been suspended due to a violation of our [Terms of Service]

I have contacted support on numerous instances to find out what has happend and how to rectify the problem.

Its extremely frustrating and unprofessional that they will not respond.

What does it take to get a response from these guys?!!

Please help

**Comment by queenofcorgis** [comment]
Hi @KevinHelton, 

Thank you for your patience, I totally understand how frustrating this situation is. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Comment by KevinHelton** [comment]
Hi Lili

Thank you for coming back to me on this.

I have been waiting patiently for a response for almost a year!

You guys are a billion dollar organization.

You must do better than this



Kevin Helton
Founder & Creative Director

LOT SIX
62—64 Southampton Row
WC1B 4AR
London

lotsix.org  |  hellolove.org
